app/src/main/res/layout/fragment_dashboard_cards.xml

```diff
@@ -111,65 +164,15 @@
                 app:layout_constraintTop_toBottomOf="@id/dash_card_positive_test" />
 
-            <cz.covid19cz.erouska.ui.dashboard.DashboardCardView
-                android:id="@+id/dash_bluetooth_off"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="8dp"
-                app:card_actionable_content="@{false}"
-                app:card_has_content="@{true}"
-                app:card_actionable_button="@{true}"
-                app:card_alert="@{true}"
-                app:card_title="@{@string/bt_disabled_title}"
-                app:card_subtitle="@{@string/bt_disabled_desc}"
-                app:card_icon="@{@drawable/ic_off_bluetooth}"
-                app:card_button_text="@{@string/enable_bluetooth_button}"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@id/dash_travel" />
-
-            <cz.covid19cz.erouska.ui.dashboard.DashboardCardView
-                android:id="@+id/dash_location_off"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="8dp"
-                app:card_actionable_content="@{false}"
-                app:card_has_content="@{true}"
-                app:card_actionable_button="@{true}"
-                app:card_alert="@{true}"
-                app:card_title="@{@string/location_off_header}"
-                app:card_subtitle="@{@string/location_off_body}"
-                app:card_icon="@{@drawable/ic_off_location}"
-                app:card_button_text="@{@string/location_off_turn_on}"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@id/dash_bluetooth_off" />
-
-            <cz.covid19cz.erouska.ui.dashboard.DashboardCardView
-                android:id="@+id/dash_bluetooth_location_off"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="8dp"
-                app:card_actionable_content="@{false}"
-                app:card_has_content="@{true}"
-                app:card_actionable_button="@{true}"
-                app:card_alert="@{true}"
-                app:card_title="@{@string/location_bt_off_header}"
-                app:card_subtitle="@{@string/location_bt_off_body_1 + `\n` + @string/location_bt_off_body_2}"
-                app:card_icon="@{@drawable/ic_off_location_bluetooth}"
-                app:card_button_text="@{@string/location_bt_off_turn_on}"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@id/dash_location_off" />
-
             <TextView
```

Comment:
will look into it but as of now, since the travel card is the last one, there shouldn't be any problem, but you are right. I thought it was expected to have it like this.